GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -448,7 +448,9 @@ public override string InstallerExtension
             {
                 get { return ".exe"; }
             }
-
+
+            public override bool SupportsInlineUpgrade { get; } = false;
```

Comment:
a few lines in comment would help understand what an `inline` upgrade is.

Reply:
If you (or anyone) has a better term for this, please suggest it :)

Reply:
Maybe `SupportsUpgradeWhileRunning` that's the best I can come up with. Not sure how accurate that is either?

Reply:
I vote for `SupportsInlineUpgrade`. The context can be made clear with additional comments. Alternate name ideas -`SupportsSingleWindowUpgrade`?

Reply:
I tweaked the name and added comments to the base abstract property.